pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/model/EventLogger.java

```diff
@@ -15,20 +17,13 @@
  */
 public class EventLogger {
 
-    private ObservableList<LogEntry> log = FXCollections.observableArrayList();
-
+    private Var<LogEntry> latestEvent = Var.newSimpleVar(null);
 
     public void logEvent(LogEntry event) {
-        log.add(event);
+        latestEvent.setValue(event);
     }
 
-
-    /**
-     * Gets an observable view of the log.
-     *
-     * @return The log
-     */
-    public ObservableList<LogEntry> getLog() {
-        return log;
+    public EventStream<LogEntry> getLog() {
```

Comment:
Why remove the doc comment? I think, it's event more important now to document it